R/github-actions.R

```diff
@@ -133,7 +95,11 @@ use_github_action <- function(name,
   use_dot_github(ignore = ignore)
 
   if (is.null(save_as)) {
-    save_as <- path_file(url)
+    if (is_check_action(url)) {
```

Comment:
This feels a bit icky — `use_github_action()` was previously an internal helper that happened to have a really good name. And now I've squished in the naming and badge logic from the older helpers.

But I think the net effect of this is neutral since you probably don't want multiple `R CMD check` workflows in one repo. (And if you do, you can use `save_as`)